src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/view-cache/view-cache.component.html

```diff
@@ -0,0 +1,14 @@
+<alert type="info"
+       *ngIf="status === vcs.ValueNone">
+  Still working, please wait.
+</alert>
+
+<alert type="warning"
+       *ngIf="status === vcs.ValueStale">
+  This information is stale.
```

Comment:
The data returned with STALE status, means that the data might be stale. It does not mean that it is stale for sure.
Please rewrite this sentence to reflect that.

Reply:
done.